.clang-format-ignore

```diff
@@ -13,3 +13,6 @@ FEXCore/Source/Interface/Core/X86Tables/*
 # Inline headers with list-like content that can't be processed individually
 Source/Tools/LinuxEmulation/LinuxSyscalls/x*/SyscallsNames.inl
 Source/Tools/LinuxEmulation/LinuxSyscalls/x*/Ioctl/*.inl
+
+# cephes math library
+FEXCore/Source/Common/cephes/*
```

Comment:
Only nit seems to be that this can be removed

Reply:
Oop, removed.